src/components/Button/Button.js

```diff
@@ -4,12 +4,12 @@ import { useTheme } from '@material-ui/core';
 
 import StyledButton from './Style';
 
-export default function CommonButton({ children, color }) {
+export default function CommonButton({ children, color, onClick, minwidth }) {
   const theme = useTheme();
 
   // One option to get theme and supply it to styled compoenents
   return (
-    <StyledButton color={color} theme={theme} variant="contained">
+    <StyledButton color={color} minwidth={minwidth} onClick={onClick} theme={theme} variant="contained">
```

Comment:
What if onClick,minWidth and color props are null the component might broke please use default props .. and also add the package of propTypes so that common component can have proper types defined for props so that one who is using it can get the idea of what a prop value must be